app/src/main/java/homer/common/time/Clock.java

```diff
@@ -0,0 +1,28 @@
+package homer.common.time;
+
+import java.time.LocalDateTime;
+import java.time.temporal.TemporalAmount;
+
+public interface Clock {
+
+    /**
+     * 
+     * @return The current local date and time of the clock.
+     */
+    LocalDateTime getDateTime();
+
+    /**
+     * Sets the clock to a new local date and time.
+     * 
+     * @param dateTime The new local date and time to set.
+     */
+    void setDateTime(LocalDateTime dateTime);
+
+    /**
+     * Updates the clock local date and time with a delta time amount.
+     * 
+     * @param deltaTime How much time should be added to the current clock time.
```

Comment:
what is the time's measure unit? i think it should be specified in the interface

Reply:
Do you mean I have to specify it's a [TemporalAmount](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/temporal/TemporalAmount.html)?
You can give for eg. a [Duration](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html) or [Period](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Period.html), which can be created in several ways.

Reply:
@michelececcacci not sure what you meant

Reply:
oh, ok, perfect. there was a misunderstanding on my side.